src/redux/reducers/index.js

```diff
@@ -0,0 +1,168 @@
+const initialState = {
+  products: [],
+  cart: [],
+  totalAmount: 0,
+  filteredProducts: [],
+  filters: {
+    category: [],
+    priceUnder: [],
+    sortOrder: null,
+  },
+  address: null,
+  theme: "light",
+};
+
+const reducer = (state = initialState, action) => {
+  switch (action.type) {
+    case "ADD_TO_CART": {
+      const itemId = parseInt(action.id);
+      const itemInCart = state.cart.find((ele) => ele.id === itemId);
+
+      if (itemInCart) {
+        const updatedCart = state.cart.map((ele) =>
+          ele.id === itemId ? { ...ele, cartCount: ele.cartCount + 1 } : ele
```

Comment:
@arun-121 ca you give a relevant name for this

Reply:
renamed as cartItem